Client/Frontend/Reader/ReaderModeStyleViewController.swift

```diff
@@ -5,78 +5,123 @@
+    static let FontSizeLabelFontSerif = UIFont(name: "Charis SIL", size: 22)
+    static let FontSizeLabelColor = UIColor(rgb: 0x333333)
+    static let FontSizeButtonTextColorEnabled = UIColor(rgb: 0x333333)
+    static let FontSizeButtonTextColorDisabled = UIColor.lightGrayColor() // TODO THis needs to be 44% of 0x333333
+    static let FontSizeButtonTextFont = UIFont(name: "FiraSans-Book", size: 22)
+
+    static let ThemeRowBackgroundColor = UIColor.whiteColor()
+    static let ThemeTitleFontSansSerif = UIFont(name: "FiraSans-Light", size: 15)
+    static let ThemeTitleFontSerif = UIFont(name: "Charis SIL", size: 15)
+    static let ThemeTitleColorLight = UIColor(rgb: 0x333333)
+    static let ThemeTitleColorDark = UIColor.whiteColor()
+    static let ThemeTitleColorSepia = UIColor(rgb: 0x333333)
+    static let ThemeBackgroundColorLight = UIColor.whiteColor()
+    static let ThemeBackgroundColorDark = UIColor.blackColor()
+    static let ThemeBackgroundColorSepia = UIColor(rgb: 0xf0ece7)
+
+    static let BrightnessRowBackground = UIColor(rgb: 0xf4f4f4)
+    static let BrightnessSliderTintColor = UIColor(rgb: 0xe66000)
+    static let BrightnessSliderWidth = 140
+    static let BrightnessIconOffset = 10
+}
+
+// MARK: -
+
 protocol ReaderModeStyleViewControllerDelegate {
     func readerModeStyleViewController(readerModeStyleViewController: ReaderModeStyleViewController, didConfigureStyle style: ReaderModeStyle)
 }
 
+// MARK: -
+
 class ReaderModeStyleViewController: UIViewController {
     var delegate: ReaderModeStyleViewControllerDelegate?
     var readerModeStyle: ReaderModeStyle = DefaultReaderModeStyle
 
-    private let rowHeights = [72, 44, 44, 52]
-    private let width = 260
-
-    private var rows: [UIView] = []
     private var fontTypeButtons: [FontTypeButton]!
+    private var fontSizeLabel: FontSizeLabel!
     private var fontSizeButtons: [FontSizeButton]!
     private var themeButtons: [ThemeButton]!
 
     override func viewDidLoad() {
-        view.backgroundColor = UIColor.grayColor()
-
         // Our preferred content size has a fixed width and height based on the rows + padding
 
-        let height = rowHeights.reduce(0, combine: +) + (rowHeights.count - 1)
-        preferredContentSize = CGSize(width: width, height: height)
-
-        // Setup the rows. It is easier to setup a layout like this when we organize the buttons into rows.
+        preferredContentSize = CGSize(width: ReaderModeStyleViewControllerUX.Width, height: ReaderModeStyleViewControllerUX.Height)
 
-        let fontTypeRow = UIView(), fontSizeRow = UIView(), themeRow = UIView(), sliderRow = UIView()
-        rows = [fontTypeRow, fontSizeRow, themeRow, sliderRow]
+        popoverPresentationController?.backgroundColor = ReaderModeStyleViewControllerUX.FontTypeRowBackground
 
-        for (idx, row) in enumerate(rows) {
-            view.addSubview(row)
+        // Font type row
```

Comment:
This function is long enough it might be nice to split up. i.e.

var row = addFontTypeRow()
row = addFontSizeRow(below: row);
row = addFontThemeRow(below: row);
addBrightnessRow(below: row);

would be slightly easier on the eyes. Given that they all start with the same code, it could also probably be refactored out. i.e.

addRow(below: view) -> UIView {
  let row = UIView()
  view.addSubview(row)
  row.snp_makeConstraints(...)
  return row
}